turn_alerts/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+from sidekick.models import Organization
+
+
+class TurnAlerts(models.Model):
```

Comment:
What's the reason for saving this?

Reply:
The reason I have this saved is so it can be configurable. Do we want to hardcode the journey_id etc or have it easily changed.

Reply:
Also, we no longer need at least half of the serializers there since we no longer need most of the key values for prometheus. I left them there just in case we needed to capture more data. I could delete most of them if you think otherwise. @erikh360

Reply:
Yes please remove all serializers we don't need.

Sorry I misunderstood this model. Lets chat in Slack